remoteappmanager/static/js/admin/vue-components/toolkit/DataTable.js

```diff
@@ -0,0 +1,52 @@
+define([
+], function() {
+  "use strict";
+
+  return {
+    props: [
+      "headers", "rows", "globalActions", "rowActions"
+    ],
+    methods: {
+      isBoolean: function(value) {
+        return typeof(value) === "boolean";
+      },
+      buttonClassFromType: function(value) { 
+        var cls = {"btn": true};
+        if (value === undefined) {
```

Comment:
With ES6 we will use a default value like in python `function(value = "danger"){...}`